### ISSUE TITLE
Build failure on FreeBSD 11-RELEASE.

### ISSUE THREAD

**Issue by vifino** [jwilm/alacritty#571]
`rustc --version`: `rustc 1.18.0-nightly (94e884b63 2017-04-27)`
```
   Compiling alacritty v0.1.0 (file:///usr/home/vifino/src/alacritty)
error[E0432]: unresolved import `libc::TIOCSCTTY`
  --> src/tty.rs:24:59
   |
24 | use libc::{self, winsize, c_int, pid_t, WNOHANG, SIGCHLD, TIOCSCTTY};
   |                                                           ^^^^^^^^^ no `TIOCSCTTY` in the root

error: aborting due to previous error

error: Could not compile `alacritty`.
```

Always reproducable.

**Comment by vifino** [comment]
I've got it compiling and pretty much working.

However, job control and resizing does not seem to work.

```
bash: cannot set terminal process group (56540): Inappropriate ioctl for device
bash: no job control in this shell
```

It's a big step up from "not building at all", at least.

**Comment by jwilm** [comment]
@vifino sure!

Strange bugs you're seeing. I wonder if the resize issue is due to a bad `ioctl` argument?

**Comment by vifino** [comment]
Fixed by f051e68, much cleaner. My PR was working around a bug in the libc crate.
